packages/utilities/src/Customizer.tsx

```diff
@@ -85,7 +85,7 @@ export class Customizer extends BaseComponent<ICustomizerProps> {
     return (
       <CustomizerContext.Consumer>
         {(parentContext: ICustomizerContext) => {
-          const newContext = this._getCustomizations(this.props, parentContext);
+          const newContext = mergeCustomizations(this.props, parentContext);
```

Comment:
Random aside: does any of this fix the issue with React's default props feature overwriting props that come down through Customizer?

Reply:
I don't think so.. in developing this feature I knew I need to merge customizations when `scheme` is provided. I found this implementation already present and the change to this file simply exposes it for use without changing any existing functionality.